packages/replay-internal/src/util/sendReplay.ts

```diff
@@ -36,8 +35,8 @@ export async function sendReplay(
       _retryCount: retryConfig.count,
     });
 
-    if (DEBUG_BUILD && options._experiments && options._experiments.captureExceptions) {
-      captureException(err);
+    if (onError) {
+      onError(err);
```

Comment:
I wanted to propagate this up so that we're always using the same error capturing logic (e.g. `replay.handleException`)